spec.emu

```diff
@@ -258,6 +291,7 @@ contributors: Ujjwal Sharma, Younies Mahmoud
         1. Let _df_ be *this* value.
         1. Perform ? RequireInternalSlot(_df_, [[InitializedDurationFormat]]).
         1. Let _record_ be ? ToPartialDuration(_duration_).
+        1. Let _record_ be ? RoundDurationComplete(_record_, _df_.[[SmallestUnit]], _df_.[[LargestUnit]], `undefined`, 1, `"halfExpand"`).
```

Comment:
What is RoundDurationComplete ?  I cannot find such operation in https://tc39.es/proposal-temporal/ nor in this spec.